### PR TITLE
Refer: #4x9t1y Bug: sent correctly the time of the creation of the or…

### PR DESCRIPTION
**Pull request by davidmaring7**
## Refer: 

- Task: https://app.clickup.com/t/4x9t1y

## Why I'm adding these lines of code?

- Because we need to correctly send the order creation time in the seller's store for the suite display.

## Before

image

## After

Captura de pantalla de 2021-06-29 18-24-18

## Message to the "nuevasfuncionalidades" channel

- (corrección) [#4x9t1y](https://app.clickup.com/t/4x9t1y) Agregar fecha creación seller en ventas.

### CODE REVIEW

File: shipit/tags/3.3.2/readme.txt [hunk 1]
```diff
@@ -0,0 +1,279 @@
+=== Shipit ===
+Contributors: franciscoarenasp
+Donate link: https://shipit.cl/
+Tags: shipping, calculator, couriers, Shipit
+Requires at least: 4.4
+Tested up to: 5.6
+Requires PHP: 5.6
+Stable tag: 3.3.1
+Version: 3.3.1
```
Comment (Zeaneth) [inline shipit/tags/3.3.2/readme.txt:9]:
```suggestion
Stable tag: 3.3.2
Version: 3.3.2
```

File: shipit/trunk/readme.txt [hunk 2]
```diff
@@ -54,6 +54,10 @@ Sí, con las acciones en lote, más información en el siguiente link: https://s
 
 == Changelog ==
 
+= 3.3.2 =
```
Comment (Zeaneth) [inline shipit/trunk/readme.txt:57]: Update stable version on lines 8 and 9

**Review by Zeaneth — CHANGES_REQUESTED** [review]

File: shipit/trunk/readme.txt (hunk 2 above)
Comment (davidmaring7) [reply shipit/trunk/readme.txt:57]: Done. @Zeaneth :+1:

**Comment by multinauta** [comment]
on line 5 of shipit.php update the version

**Review by multinauta — APPROVED** [review]

**Review by tostitos-dev — CHANGES_REQUESTED** [review]
Include message to the "nuevasfuncionalidades" channel

**Review by gcamposm — APPROVED** [review]

**Review by Barbycris — APPROVED** [review]
You never assign yourself to the PR 🤔

**Review by luispascuali — APPROVED** [review]

**Review by tostitos-dev — APPROVED** [review]

**Review by Zeaneth — COMMENTED** [review]
the branch's name is not up to our standards (with an underscore `_` instead of `-`).

**Review by Zeaneth — APPROVED** [review]

**Pull request merged**